scripts/linters/html_linter_test.py

```diff
@@ -129,6 +133,26 @@ def test_custom_without_html_end_tag(self):
                 [INVALID_MISSING_HTML_TAG_HTML_FILEPATH], FILE_CACHE
                 ).perform_all_lint_checks()
 
+    def test_custom_space_around_html_attribute(self):
+        lint_task_report = html_linter.HTMLLintChecksManager(
+            [INVALID_SPACE_AROUND_ATTRIBUTE_HTML_FILEPATH], FILE_CACHE
+        ).check_html_tags_and_attributes()
+        self.assert_same_list_elements([
+            'Attribute class for tag div on line 4 ',
+            'has unwanted white spaces around it'
+        ], lint_task_report.trimmed_messages)
+        self.assertTrue(lint_task_report)
+
+    def test_custom_space_innerhtml_attribute(self):
```

Comment:
`test_space_around_attribute_having_camleCase_name_reports_correctly` (Change the test name similarly above*)

Reply:
done, ive named it `test_space_around_attr_having_camel_case_name_reports_correctly` since the name you suggested is more than 80 characters.